vm/src/stdlib/os.rs

```diff
@@ -221,6 +221,31 @@ fn convert_nix_errno(vm: &VirtualMachine, errno: Errno) -> PyClassRef {
     }
 }
 
+fn os_access(path: PyStringRef, mode: PyIntRef, vm: &VirtualMachine) -> PyResult {
```

Comment:
You can return a `PyResult<bool>` to avoid having to call `vm.new_bool` yourself.

Reply:
I did that first but I wasn sure it would send it to the context.

Reply:
It does, `bool` implements the `IntoPyObject` trait that I mentioned in my other comment, and `IntoPyObject` is implemented for `PyResult<T> where T: IntoPyObject`.